### ISSUE TITLE
Output HTML is beautified, causing rendering differences

### ISSUE THREAD

**Issue by calmyournerves** [ViewComponent/lookbook#470]
### Describe the bug

After upgrading from 2.0 we noticed differences in the HTML output of some previews. It seems the HTML output of the rendered components is formatted with the `CodeBeautifier` class which beautifies the outputted HTML, causing rendering differences.

When commenting out the [call to `CodeBeautifier` in `RenderedScenarioEntity`](https://github.com/ViewComponent/lookbook/blob/main/lib/lookbook/entities/rendered_scenario_entity.rb#L27) it seems to work as expected.

### To Reproduce

1. Create a preview with the following HTML
```HTML
<div><div class="inline-block border">Test</div><div class="inline-block border">Test</div><div class="inline-block border">Test</div></div>

<div>
  <div class="inline-block border">Test</div>
  <div class="inline-block border">Test</div>
  <div class="inline-block border">Test</div>
</div>
```
2. Open the preview and check the source, which is now:
```HTML
<div>
  <div class="inline-block border">Test</div>
  <div class="inline-block border">Test</div>
  <div class="inline-block border">Test</div>
</div>
<div>
  <div class="inline-block border">Test</div>
  <div class="inline-block border">Test</div>
  <div class="inline-block border">Test</div>
</div>
```

And looks like this (note the spaces between the first row of elements):
![image](https://github.com/ViewComponent/lookbook/assets/480921/07149ee5-4b0c-4dae-b08e-cfa8e87af2cd)

### Expected behavior

In 1.5.5, the source formatting was respected, resulting in the correct output:

```HTML
<div><div class="inline-block border">Test</div><div class="inline-block border">Test</div><div class="inline-block border">Test</div></div>

<div>
  <div class="inline-block border">Test</div>
  <div class="inline-block border">Test</div>
  <div class="inline-block border">Test</div>
</div>
```

Which looks like this (no spaces between the elements in the first row):
![image](https://github.com/ViewComponent/lookbook/assets/480921/811b4122-c4d6-4065-aba1-4a14331676c3)

### Version numbers 

Please complete the following information:

* Lookbook: 2.0.2
* ViewComponent: 2.74.1
* Rails: 7.0.4.3
* Ruby: 3.1.2

**Comment by allmarkedup** [comment]
@calmyournerves this should now be fixed in v2.0.3 which I've just released. I'm going to go ahead and close this now but if you are still having problems feel free to reopen or let me know!

Thanks again for taking the time to open an issue and provide code to help reproduce it - much appreciated.

**Comment by allmarkedup** [comment]
@calmyournerves Fantastic, thanks for letting me know, and my pleasure! :-)
